excel/src/main/java/org/apache/metamodel/excel/ExcelConfiguration.java

```diff
@@ -102,17 +115,39 @@ public boolean isSkipEmptyColumns() {
 		return skipEmptyColumns;
 	}
 
+    /**
+     * Defines if columns in the excel spreadsheet should be validated on datatypes while
```

Comment:
Instead of "validated" we should probably write "detected". And we should also say what happens when this is `false` - then all columns will be of STRING type.

Reply:
I've changed this JavaDoc. Do note that in order to stay backwards compatible it's either String or VarChar depending on the Column Name Line configuration. I try to explain the differences in the JavaDoc.
I think we should only return String soon. The current situation is hard to understand and very confusing.